homeassistant/components/script/blueprints/random_color.yaml

```diff
@@ -0,0 +1,109 @@
+blueprint:
```

Comment:
This script is a good example of what is possible, but it doesn't feel useful enough that I would expect more than 10% of our users to adopt it. 

Can we think of a script that would be more popular and so deserves more to be one that we copy to their config folder?

What about a script that sends an actionable notification to mobile app and executes the blueprinted action if the user approves? Selector is mobile device + action. Tied together with a wait_for_trigger.

Reply:
As discussed in Discord, awaiting some changes to iOS. In the meantime, I've removed the population of built-in Blueprints for scripts. Additionally, simplified the one that is in there a bit (removed default target). Kept it, as it is used for tests.

Reply:
Shouldn't we move it then to the testing dir? If I recall correctly, we're automatically copying `components/<component name>/blueprints` to the user's dir if the user has none. Or do you mean we shouldn't merge it until iOS is fixed?

Reply:
That logic is separated, I simply removed that method call that does that, until we have something we want to popluate.

See https://github.com/home-assistant/core/commit/837afa3ef602fd45d2ad4950913625de8bc1fe09

Reply:
Later if we have a good one, we can add it, call populate and adjust the tests

Reply:
Restored it.